debug.md

```diff
@@ -0,0 +1,58 @@
+# Debugging
+
+In cases where you need to debug Ledger communication, you may need:
+
+1. Debug Firmware on the Ledger Device
+2. A debug version of the Nervos Ledger App
+3. Monitored communication between the device and the client via usbtool
+
+## Debugging on NixOS
+
+### Installing Debug Ledger Firmware
+
+todo
```

Comment:
You can get the ledgerblue python module in scope with just `nix/env.sh s` and then follow the steps like you do for the mac. So it'd be "download the firmware image, nix/env.sh s, run the python commands to send it to the ledger".

Reply:
Thanks! I incorporated this feedback in the latest commit